src/pss/TopicSubject.js

```diff
@@ -50,6 +55,17 @@ export class TopicSubject extends AnonymousSubject<Object> {
     log('setup')
   }
 
+  async sendMessageToPeers(data: TopicMessageEvent): Promise<*> {
```

Comment:
Not an issue but the return type could be explicitly set to `Promise<void>`, no need to use the inference.